frontend/src/components/MultiSelect.js

```diff
@@ -165,35 +166,38 @@ function MultiSelect({
       render={({ onChange: controllerOnChange, value, onBlur }) => {
         const values = value ? getValues(value) : value;
         return (
-          <Selector
-            className="ttahub-multi-select margin-top-1"
-            id={name}
-            value={values}
-            onBlur={onBlur}
-            onChange={(event) => {
-              if (onItemSelected) {
-                onItemSelected(event);
-              } else if (event) {
-                onChange(event, controllerOnChange);
-              } else {
-                controllerOnChange([]);
-              }
-            }}
-            inputId={inputId}
-            styles={styles(singleRowInput)}
-            components={{ ...componentReplacements, DropdownIndicator }}
-            options={options}
-            isDisabled={disabled}
-            tabSelectsValue={false}
-            isClearable={multiSelectOptions.isClearable}
-            closeMenuOnSelect={multiSelectOptions.closeMenuOnSelect || false}
-            controlShouldRenderValue={multiSelectOptions.controlShouldRenderValue}
-            hideSelectedOptions={multiSelectOptions.hideSelectedOptions}
-            placeholder={placeholderText || ''}
-            onCreateOption={onCreateOption}
-            isMulti
-            required={!!(required)}
-          />
+          // eslint-disable-next-line jsx-a11y/no-static-element-interactions
+          <div onClick={onClick} onKeyDown={() => {}} data-testid={`${name}-click-container`}>
```

Comment:
Should we have onKeyDown with an empty func().

Reply:
I think we probably want to watch for the enter key